downstream/modules/eda/proc-eda-view-rule-audit-events.adoc

```diff
@@ -0,0 +1,12 @@
+[id="eda-view-rule-audit-events"]
+
+= Viewing rule audit events
+
+.Procedure
+
+. From the navigation panel select btn:[Rule Audit].
+. Select the desired rule, this brings you to the *Details* tab.
+This shows you the event that triggered an action.
```

Comment:
I think this and the above line should say something more along the lines of:
"Select the desired rule, this brings you to the Details tab. To view all the events that triggered an action, select the Events tab."
Feel free to have Alejandro confirm as well. His account doesn't show up when I try to tag him.

Reply:
Agree, just a detail: the action should be always plural, "actions",  it might be more than one per rule.

Reply:
Done :+1: